packages/lit-node-client/src/lib/lit-node-client.ts

```diff
@@ -1344,28 +1430,33 @@ export class LitNodeClient {
 
     if (!paramsIsSafe) return;
 
-    // ========== Prepare Variables ==========
-    // -- prepare request body
-    const reqBody: JsonExecutionRequest = this.getLitActionRequestBody(params);
-
-    // ========== Get Node Promises ==========
-    // -- fetch shares from nodes
-    const nodePromises = this.getNodePromises((url: string) => {
-      // -- choose the right signature
-      let sigToPassToNode = this.getAuthSigOrSessionAuthSig({
-        authSig,
-        sessionSigs,
-        url,
-      });
-      reqBody.authSig = sigToPassToNode;
+    let res;
+    if (singleNode) {
```

Comment:
feel free to restructure this.  but i figured both single and multi execution can return an array of response promises so it makes sense to put the if statement here